hybrid/HybridSampleApps/AccountEditor/AndroidManifest.xml

```diff
@@ -20,7 +20,8 @@
         <!--  Main activity -->
         <activity android:label="@string/app_name"
             android:name="com.salesforce.androidsdk.phonegap.ui.SalesforceDroidGapActivity"
-            android:configChanges="orientation|keyboardHidden">
+            android:configChanges="orientation|keyboardHidden"
+            android:theme="@style/Theme.AppCompat.DayNight">
```

Comment:
We need this theme since they now extend `AppCompatActivity` instead of `Activity`.